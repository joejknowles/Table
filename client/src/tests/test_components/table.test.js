import Table from '../../components/table';
import React from 'react';
import ReactDOM from 'react-dom';

import ReactTestUtils from 'react-addons-test-utils';
const shallowRenderer = ReactTestUtils.createRenderer();

it('component renders as before', () => {
  const component = shallowRenderer.render(<Table />);
  expect(component).toMatchSnapshot();
});
