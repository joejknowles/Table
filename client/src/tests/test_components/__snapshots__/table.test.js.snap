exports[`test component renders as before with top card 1`] = `
<Unknown>
  <div
    className="pile">
    <div
      className="card">
      undefined undefined
    </div>
  </div>
</Unknown>
`;

exports[`test component renders as before without top card 1`] = `
<Unknown>
  <div
    className="pile">
    <div
      className="noCards" />
  </div>
</Unknown>
`;
